src/contour/helper.cpp

```diff
@@ -436,8 +436,10 @@ void sendWheelEvent(QWheelEvent* event, TerminalSession& session)
     //       it will send horizontal wheel events instead of vertical ones. We need to compensate
     //       for that here.
 
+    auto const dpr = session.display()->contentScale();
+
     auto const pixelDelta =
-        (modifiers & Modifier::Alt) ? transposed(event->pixelDelta()) : event->pixelDelta();
+        dpr * ((modifiers & Modifier::Alt) ? transposed(event->pixelDelta()) : event->pixelDelta());
```

Comment:
Maybe something like this slightly better for readability
```
auto transform = [&](auto delta){ 
return (modifiers & Modifier::Alt) ?  transposed(delta) : delta;
};

auto const pixelDelta = dpr * transform(event->pixelDelta());
```

Reply:
Not sure the right hand side is a transform but rather an identity, but I'll refactor :)

Reply:
@Yaraslaut check again, please